q2_alignment/plugin_setup.py

```diff
@@ -33,7 +36,17 @@
     parameters={'max_gap_frequency': Float,
                 'min_conservation': Float},
     outputs=[('masked_alignment', FeatureData[AlignedSequence])],
+    input_descriptions={'alignment': 'The alignment to be masked.'},
+    parameter_descriptions={
+        'max_gap_frequency': ('The maximum fraction of sequences that can '
```

Comment:
You're using "column" here and "position" below. Can you pick one term? I don't have a preference.